torch/_inductor/triton_heuristics.py

```diff
@@ -1025,6 +1028,9 @@ def triton_config_tiled_reduction(size_hints, x, y, r, num_stages=1):
     cfg = {"XBLOCK": x, "YBLOCK": y, "RBLOCK": r}
     num_warps = next_power_of_2(min(max(conditional_product(x, y, r) // 256, 1), 8))
     check_config(cfg, xnumel=size_hints[0], ynumel=size_hints[1])
+    assert (
```

Comment:
Is this function `triton_config_tiled_reduction` even called somewhere?

Reply:
No